src/msgpack.hrl

```diff
@@ -37,9 +37,9 @@
           spec = new :: new | old,
```

Comment:
1. Shouldn't we update `msgpack_term()` spec to include tagged lists/strings?
2. Someone forgot to add `positive_infinity` and the like to the spec earlier.